server/controllers/user.js

```diff
@@ -222,6 +277,20 @@ const createCandidate = async (req, res) => {
 };
 
 const vote = async (req, res) => {
+  const voteSchema = schema({
```

Comment:
What does the vote schema have to do with a password reset?

Reply:
While working on the reset password feature, I also made some changes to other controllers.
Is it a bad practice? Should I only make changes on the specific things I am working on?